users/users-model.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../data/db-config.js');
+
+function find() {
+  return db('users').select('id', 'username').orderBy('id');
+}
+
+function findBy(username) {
```

Comment:
maybe use a findByFilter....can be more diverse and allow for accessing other columns in the DB :D